server/src/app.ts

```diff
@@ -22,5 +23,6 @@ app.use(userRouter);
 app.use(feedRouter);
 app.use(projectRouter);
 app.use(milestoneRouter);
+app.use(documentRouter);
```

Comment:
I'm not sure... but it seems that the article router missing...

Reply:
it absolutely is, i'll add it right away